src/main/java/dataloader/JsonDataLoader.java

```diff
@@ -0,0 +1,73 @@
+package dataloader;
+
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import tech.tablesaw.api.Table;
+import tech.tablesaw.io.json.JsonReadOptions;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+/**
+ * Class that takes a String as path
```

Comment:
It's not the class that takes the String :) :) I think sth along the lines ```"class JsonDataLoader loads data from a specified json file...."```  the rest as you have it....I think is better

Reply:
done!